bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Znunugamma_aTGC/Znunugamma_aTGC_ZZg_h3Z_h4Z_Ptg150to600_2jets/Znunugamma_aTGC_ZZg_h3Z_h4Z_Ptg150to600_2jets_run_card.dat

```diff
@@ -0,0 +1,253 @@
+#*********************************************************************
+  0	=  ptl1min  ! minimum pt for the leading lepton in pt
+  0	=  ptl2min  ! minimum pt for the second lepton in pt
+  0	=  ptl3min  ! minimum pt for the third lepton in pt
+  0	=  ptl4min  ! minimum pt for the fourth lepton in pt
+  -1	=  ptl1max  ! maximum pt for the leading lepton in pt 
+  -1	=  ptl2max  ! maximum pt for the second lepton in pt
+  -1	=  ptl3max  ! maximum pt for the third lepton in pt
+  -1	=  ptl4max  ! maximum pt for the fourth lepton in pt
+#*********************************************************************
+# Control the Ht(k)=Sum of k leading jets                            *
+#*********************************************************************
+  0	=  htjmin  ! minimum jet HT=Sum(jet pt)
+  -1	=  htjmax  ! maximum jet HT=Sum(jet pt)
+  0	=  ihtmin   !inclusive Ht for all partons (including b)
+  -1	=  ihtmax   !inclusive Ht for all partons (including b)
+  0	=  ht2min  ! minimum Ht for the two leading jets
+  0	=  ht3min  ! minimum Ht for the three leading jets
+  0	=  ht4min  ! minimum Ht for the four leading jets
+  -1	=  ht2max  ! maximum Ht for the two leading jets
+  -1	=  ht3max  ! maximum Ht for the three leading jets
+  -1	=  ht4max  ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+  0	=  ptgmin  ! Min photon transverse momentum
+  0.4	=  R0gamma  ! Radius of isolation code
+  1.0	=  xn  ! n parameter of eq.(3.4) in hep-ph/9801442
+  1.0	=  epsgamma  ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+  .true.	=  isoEM  ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+  0	=  xetamin  ! minimum rapidity for two jets in the WBF case  
+  0	=  deltaeta  ! minimum rapidity for two jets in the WBF case 
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+  -1	=   ktdurham        
+   0.4	=   dparameter 
+ #*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+  4	=  maxjetflavor     ! Maximum jet pdg code
```

Comment:
This is inconsistent with your setting
5	=  asrwgtflavor      ! highest quark flavor for a_s reweight

Please check your model whether it is 4f or 5f

Reply:
Hi, 
the model is 4 F
Here is output of display multiparticles 

MG5_aMC>import model NTGC_UFO
INFO: reload from .py file 
INFO: load particles 
INFO: load vertices 
INFO: Change particles name to pass to MG5 convention 
Kept definitions of multiparticles l- / j / vl / l+ / p / vl~ unchanged
Defined multiparticle all = g a ve vm vt ve~ vm~ vt~ u c t d s b u~ c~ t~ d~ s~ b~ z w+ h w- e- mu- ta- e+ mu+ ta+
MG5_aMC>display multiparticles
Multiparticle labels:
all = g a ve vm vt ve~ vm~ vt~ u c t d s b u~ c~ t~ d~ s~ b~ z w+ h w- e- mu- ta- e+ mu+ ta+
l- = e- mu-
j = g u c d s u~ c~ d~ s~
vl = ve vm vt
l+ = e+ mu+
p = g u c d s u~ c~ d~ s~
vl~ = ve~ vm~ vt~

Reply:
So it's 4f

Reply:
so it's 4f